Models/DTO/Administrator/PharmaciesReportDTO.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace API.Models.DTO.Administrator
+{
+    public class PharmaciesReportDTO
+    {
+        [JsonProperty("allamount")]
+        public decimal AllAmount { get; set; }
+
+        [JsonProperty("mostprofitablepharmacy ")]
+        public string MostProfitablePharmacy { get; set; }
+
+        [JsonProperty("numberofpharmacies ")]
+        public int NumberOfPharmacies { get; set; }
```

Comment:
It is better to avoid such long and convoluted property names. Maybe they could be clarified?

Reply:
Changed: toppharmacy and biggestamount.